### DISCUSSION TITLE
Slack notification for discussion comments

### DISCUSSION THREAD

**Discussion by kimploo** [community/community#11642] [Discussions]
Hello @evi-liu , Thank you for awesome feature.
As a coding bootcamp instructor, I uses this discussion feature as our internal "StackOverflow", called "agora-states".
Our engineers are working hard on answering students' questions. When it comes to answering questions, responsiveness is important for us to get notified via slack notification.

In case of github issue, I can do something like `/github subscribe owner/repo issues comments` to get the issue's comment. I want similar feature in discussions too! something like `/github subscribe owner/repo discussions discussionComments` would be awesome!

**Comment by Relequestual** [comment]
If you're able to use Zapier while we wait for this feature, I already did the leg work soon after hooks were avilable.
Details: https://github.com/github/feedback/discussions/2844#discussioncomment-1124302
It does the WHOLE job for Discussions though, not JUST comments, but new discussions too.

**Reply by kimploo** [reply]
Thank you for your advice! If Zapier works, AWS Lambda will work too I guess.
